packages/twenty-front/src/modules/ui/field/input/components/DateInput.tsx

```diff
@@ -32,10 +30,13 @@ export type DateInputProps = {
   isDateTimeInput?: boolean;
   onClear?: () => void;
   onSubmit?: (newDate: Nullable<Date>) => void;
+  hideHeaderInput?: boolean;
+  temporaryValue: Nullable<Date>;
+  setTemporaryValue: (newValue: Nullable<Date>) => void;
```

Comment:
Passing a setter as props is a code smell, it means either there is an inversion of layers of abstraction (child handling parent logic for example) or that we should instead use a Recoil state.